go/externals/proof_support_reddit.go

```diff
@@ -99,9 +100,18 @@ func (t RedditServiceType) FormatProofText(ppr *libkb.PostProofRes) (res string,
 
 	q := urlReencode(libkb.HTTPArgs{"title": libkb.S{Val: title}, "text": libkb.S{Val: ppr.Text}}.EncodeToString())
 
+	// The new reddit mobile site doesn't respect the post-pre-populate query
+	// parameters. Use the old mobile site until they fix this.
+	var host string
+	if libkb.IsIOS || runtime.GOOS == "android" {
```

Comment:
I saw the `DesktopAppType` constant, but it looks like that's only ever returned from `libcmdline`? I wasn't able to tell quite what was going on there. Let me know if I should try to use those constants.